packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleAnchorTabsTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Linq;
+using AngleSharp.Dom;
+using Bunit;
+using Xunit;
+
+namespace NimbleBlazor.Tests.Unit.Components;
+
+/// <summary>
+/// Tests for <see cref="NimbleAnchorTabs"/>
+/// </summary>
+public class NimbleAnchorTabsTests
```

Comment:
Add a test for validating `ActiveId` is present in DOM?

Reply:
Added